### PR TITLE
Filled missing Doc for String and FileDialog

### PR DESCRIPTION
**Pull request by AlexHolly**

### CODE REVIEW

File: doc/classes/FileDialog.xml [hunk 1]
```diff
@@ -56,6 +58,7 @@
 	</methods>
 	<members>
 		<member name="access" type="int" setter="set_access" getter="get_access" enum="FileDialog.Access">
+			See enum [code]Access[/code] constants.
```
Comment (NathanLovato) [inline doc/classes/FileDialog.xml]: I would recommend trying to explain what this does, just to get a sense for what "access" means. Especially as the constants are not documented very well at the moment.

File: doc/classes/FileDialog.xml [hunk 2]
```diff
@@ -67,13 +70,16 @@
 			The currently selected file path of the file dialog.
 		</member>
 		<member name="filters" type="PoolStringArray" setter="set_filters" getter="get_filters">
+			Set file type filter. This example shows only .png and .gd files "set_filters(PoolStringArray(["*.png","*.gd"]))".
 		</member>
 		<member name="mode" type="int" setter="set_mode" getter="get_mode" enum="FileDialog.Mode">
+			See enum [code]Mode[/code] constants.
```
Comment (NathanLovato) [inline doc/classes/FileDialog.xml]: Same here

File: doc/classes/FileDialog.xml [hunk 3]
```diff
@@ -31,12 +31,14 @@
...
 			</return>
 			<description>
+				Clear currently selected items in the dialog.
 			</description>
 		</method>
 		<method name="get_line_edit">
 			<return type="LineEdit">
 			</return>
 			<description>
+				Returns the LineEdit for the Selected File.
```
Comment (LikeLakers2) [inline doc/classes/FileDialog.xml]: I think "Selected File" should be lowercase.

File: doc/classes/FileDialog.xml [hunk 4]
```diff
@@ -67,13 +70,16 @@
 			The currently selected file path of the file dialog.
 		</member>
 		<member name="filters" type="PoolStringArray" setter="set_filters" getter="get_filters">
+			Set file type filter. This example shows only .png and .gd files "set_filters(PoolStringArray(["*.png","*.gd"]))".
```
Comment (LikeLakers2) [inline doc/classes/FileDialog.xml]: `[code]set_filters(PoolStringArray(["*.png","*.gd"]))[/code]`

File: doc/classes/FileDialog.xml (hunk 1 above)
Comment (AlexHolly) [reply doc/classes/FileDialog.xml]: I will try to explain it a bit. The constants looked like a good description to me.

**Comment by AlexHolly** [comment]
FileDialog and EditorFileDialog are very similar maybe we can use the same description on some methods?
https://github.com/godotengine/godot/blob/master/doc/classes/EditorFileDialog.xml

**Comment by akien-mga** [comment]
Cherry-picked for 3.0.7.

**Pull request merged**